auto_tune_vllm/utils/version_manager.py

```
@@ -0,0 +1,299 @@
+                
+                creation_time = datetime.fromtimestamp(file_path.stat().st_mtime)
+                
+                versions.append(VLLMDefaultsVersion(
+                    version=version_str,
+                    file_path=file_path,
+                    creation_time=creation_time,
+                    sections=sections,
+                    total_defaults=total_defaults
+                ))
+                
+            except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)